Error: (psycopg2.errors.NotNullViolation) null value in column "project_id" of relation "deworming_log" violates not-null constraint
DETAIL:  Failing row contains (8658bdf9-01d6-4a52-bacc-870be9079b1a, null, 2025-09-24, 23:09:00, 0f5df80d-b251-452d-b843-5827b05c81a0, d089c43b-f65c-40d9-9a03-94503f828f49, 4.7, thsfdhse, eahrthwrthwrthw, null, null, null, مل, فموي, null, null, إسهال, null, null, 9369f942-0185-46d5-b73b-acd1b2609c8b, 2025-09-24 18:06:44.136677, 2025-09-24 18:06:44.136681).

[SQL: INSERT INTO deworming_log (id, project_id, date, time, dog_id, specialist_employee_id, dog_weight_kg, product_name, active_ingredient, standard_dose_mg_per_kg, calculated_dose_mg, administered_amount, amount_unit, administration_route, batch_number, expiry_date, adverse_reaction, notes, next_due_date, created_by_user_id, created_at, updated_at) VALUES (%(id)s::UUID, %(project_id)s::UUID, %(date)s, %(time)s, %(dog_id)s::UUID, %(specialist_employee_id)s::UUID, %(dog_weight_kg)s, %(product_name)s, %(active_ingredient)s, %(standard_dose_mg_per_kg)s, %(calculated_dose_mg)s, %(administered_amount)s, %(amount_unit)s, %(administration_route)s, %(batch_number)s, %(expiry_date)s, %(adverse_reaction)s, %(notes)s, %(next_due_date)s, %(created_by_user_id)s::UUID, %(created_at)s, %(updated_at)s)]
[parameters: {'id': '8658bdf9-01d6-4a52-bacc-870be9079b1a', 'project_id': None, 'date': datetime.date(2025, 9, 24), 'time': datetime.time(23, 9), 'dog_id': '0f5df80d-b251-452d-b843-5827b05c81a0', 'specialist_employee_id': 'd089c43b-f65c-40d9-9a03-94503f828f49', 'dog_weight_kg': 4.7, 'product_name': 'thsfdhse', 'active_ingredient': 'eahrthwrthwrthw', 'standard_dose_mg_per_kg': None, 'calculated_dose_mg': None, 'administered_amount': None, 'amount_unit': 'مل', 'administration_route': 'فموي', 'batch_number': None, 'expiry_date': None, 'adverse_reaction': 'إسهال', 'notes': None, 'next_due_date': None, 'created_by_user_id': UUID('9369f942-0185-46d5-b73b-acd1b2609c8b'), 'created_at': datetime.datetime(2025, 9, 24, 18, 6, 44, 136677), 'updated_at': datetime.datetime(2025, 9, 24, 18, 6, 44, 136681)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
Error: (psycopg2.errors.NotNullViolation) null value in column "project_id" of relation "deworming_log" violates not-null constraint
DETAIL:  Failing row contains (678a1508-428d-48d2-aecf-5af4e33ab484, null, 2025-09-24, 23:09:00, 0f5df80d-b251-452d-b843-5827b05c81a0, d089c43b-f65c-40d9-9a03-94503f828f49, 4.7, thsfdhse, eahrthwrthwrthw, null, null, null, مل, فموي, null, null, إسهال, null, null, 9369f942-0185-46d5-b73b-acd1b2609c8b, 2025-09-24 18:07:09.69618, 2025-09-24 18:07:09.696185).

[SQL: INSERT INTO deworming_log (id, project_id, date, time, dog_id, specialist_employee_id, dog_weight_kg, product_name, active_ingredient, standard_dose_mg_per_kg, calculated_dose_mg, administered_amount, amount_unit, administration_route, batch_number, expiry_date, adverse_reaction, notes, next_due_date, created_by_user_id, created_at, updated_at) VALUES (%(id)s::UUID, %(project_id)s::UUID, %(date)s, %(time)s, %(dog_id)s::UUID, %(specialist_employee_id)s::UUID, %(dog_weight_kg)s, %(product_name)s, %(active_ingredient)s, %(standard_dose_mg_per_kg)s, %(calculated_dose_mg)s, %(administered_amount)s, %(amount_unit)s, %(administration_route)s, %(batch_number)s, %(expiry_date)s, %(adverse_reaction)s, %(notes)s, %(next_due_date)s, %(created_by_user_id)s::UUID, %(created_at)s, %(updated_at)s)]
[parameters: {'id': '678a1508-428d-48d2-aecf-5af4e33ab484', 'project_id': None, 'date': datetime.date(2025, 9, 24), 'time': datetime.time(23, 9), 'dog_id': '0f5df80d-b251-452d-b843-5827b05c81a0', 'specialist_employee_id': 'd089c43b-f65c-40d9-9a03-94503f828f49', 'dog_weight_kg': 4.7, 'product_name': 'thsfdhse', 'active_ingredient': 'eahrthwrthwrthw', 'standard_dose_mg_per_kg': None, 'calculated_dose_mg': None, 'administered_amount': None, 'amount_unit': 'مل', 'administration_route': 'فموي', 'batch_number': None, 'expiry_date': None, 'adverse_reaction': 'إسهال', 'notes': None, 'next_due_date': None, 'created_by_user_id': UUID('9369f942-0185-46d5-b73b-acd1b2609c8b'), 'created_at': datetime.datetime(2025, 9, 24, 18, 7, 9, 696180), 'updated_at': datetime.datetime(2025, 9, 24, 18, 7, 9, 696185)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj)